packages/neuron-ui/src/utils/const.ts

```diff
@@ -62,3 +62,34 @@ export enum Routes {
 export enum LocalStorage {
   Networks = 'networks',
 }
+
+export enum CapacityUnit {
+  CKB = 'ckb',
+  CKKB = 'ckkb',
+  CKGB = 'ckgb',
```

Comment:
Is this temporary unit or RFC or what is already defined by CKB?

Reply:
Just temporary uint